up-l1/zenoh.adoc

```diff
@@ -40,7 +40,17 @@ For more information, please visit https://projects.eclipse.org/projects/iot.zen
 
 === Zenoh Version
 
-We **MUST** use Zenoh version `0.10.1-rc` to ensure the interoperability in different language bindings.
+We **MUST** use Zenoh version `0.11.0-rc` to ensure the interoperability in different language bindings.
```

Comment:
@gregmedd Just want to raise it up that we'll use 0.11.0-rc in up-spec 1.5.8. I think you're using it now.